GDJS/Runtime/runtimescene.ts

```diff
@@ -732,17 +732,29 @@ namespace gdjs {
      * Tool function filling _allInstancesList member with all the living object instances.
      */
     _constructListOfAllInstances() {
+      for (const name in this._layers.items) {
+        if (this._layers.containsKey(name)) {
```

Comment:
This check is not necessary because the object of `_layers` is local to RuntimeScene.
We should remove the other checks, but I'll do it in my PR.